client/src/components/pages/SignUpParent/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import Input from '../../common/Input';
+import Button from '../../common/Button';
+import signUpValidation from '../utils/signUpValidation';
+import './index.css';
+
+export default class SignUp extends React.Component {
+  state = {
+    email: '',
+    userName: '',
+    parentId: '',
+    password: '',
+    confirmPassword: '',
+    error: false,
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const {
+      error,
+      email,
+      userName,
+      parentId,
+      password,
+      confirmPassword,
+    } = this.state;
+    signUpValidation
+      .isValid({ error, email, userName, parentId, password, confirmPassword })
```

Comment:
Why do you pass error to the validation schema? And where do you check if password is equal to confirmPassword?

Reply:
I haven't finished it yet.